common/jenkins-slaves/nodejs10-angular/Dockerfile.rhel7

```diff
@@ -62,9 +62,14 @@ RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash - && \
     yum clean all -y
 
 COPY npmrc $HOME/.npm-global/etc/npmrc
+COPY npmrc $HOME/.npmrc
+COPY yarnrc $HOME/.yarnrc
```

Comment:
yarn requires the .npmrc and .yarnrc to be directly in the home directory - else it will not work as it should (according to my tests)

Reply:
if we do the symbolic linking suggestion above we could then avoid the second `COPY npmrc $HOME/.npmrc `

Reply:
Given that Yarn expects the files in `$HOME` - is there a way to tell NPM to use that, instead of `$HOME/.npm-global/etc/npmrc`?

Reply:
I will check on this; will update you soon

Reply:
Given [this documentation](https://docs.npmjs.com/files/npmrc) we should be able to remove the global one since it is resolving project specific, then user specific and then the two more global ones.

Also it seems to be possible that we can use environment variables directly in there - so I will test if yarn can do the same - then we can get rid of some or maybe all "sed magic" for that